### DISCUSSION TITLE
Question on behavior of forecast function for multi-step horizon

### DISCUSSION THREAD

**Discussion by tszumowski** [tinkoff-ai/etna#437] [Q&A]
Suppose I wish to forecast out 8 weeks, and I have a lagged regressor that is lagged 2 steps back. When I call `make_future` on the time series, I end up with a bunch of NaN values for the lagged regressor, which makes sense. Because for example, if forecasting week 6, you have lags for weeks 4 and 5 in the future. And those aren't known ahead of time. 

However, when this is fed into the forecast, the forecaster (specifically [CatBoostMultiSegment](https://github.com/tinkoff-ai/etna/blob/9eedbd6591642687ba7853c1ae6df9b3033c129e/etna/models/catboost.py#L37-L49)) takes all inputs except `target` and `timestamp` as features to fit and predict on. 

This means there are several future forecast weeks that have a lot of missing data. I was curious why this approach was taken instead of applying a Direct Forecasting method as described in [this article](https://machinelearningmastery.com/multi-step-time-series-forecasting-with-machine-learning-models-for-household-electricity-consumption/).

**Comment by Mr-Geekman** [comment]
Thank you for the question! We are aware that we don't currently support the full Direct strategy: our approach is simplier. However, we are going to add it in the future using current approach as a starting point.
